mks/model/stuf_02_04.py

```diff
@@ -314,6 +317,57 @@ def extract_verbintenis_data(persoon_tree: Tag):
 
 def extract_address(persoon_tree: Tag, is_amsterdammer):
     result = []
+    
```

Comment:
Ik denk dat het beter is om op dit punt al te checken of je adresgegevens binnenkrijgt.

```
    addresses_vbl_source = persoon_tree.find_all("PRSADRVBL")
    addresses_cor_source = persoon_tree.find_all("PRSADRCOR")

    if is_nil(addresses_cor_source) and is_nil(addresses_cor_source):
        return {}, []
```